www/templates/home.html

```diff
@@ -1,20 +1,12 @@
-<ion-view view-title="Request a Valet">
-  <ion-content data-tap-disabled="true">
-    <form class="item item-input" ng-submit="sendPickupLocation(query)">
-      <i class="icon ion-search placeholder-icon"></i>
-      <input type="text" ng-model="query" placeholder="Search">
-    </form>
-    {{mapConfig}}
+<ion-view view-title="{{addressDisplay}}">
+  <ion-content data-tap-disa.led="true" scroll="false">
+    <img id="center_pin" src="img/center_pin.svg.png" height="30px">
```

Comment:
if the height is going to be 30 pix, why don't you resize the image to 30 pix height? ;)